packages/katello/katello-repos/katello-repos.spec

```diff
@@ -3,12 +3,11 @@
 %define repo_dir %{_sysconfdir}/yum.repos.d
 %define repo_dist %{dist}
 
-%global prereleasesource nightly
-%global prerelease %{?prereleasesource:.}%{?prereleasesource}
-%global release 3
+%global prerelease .rc1
```

Comment:
Technically correct, but obal would have generated a prerelease source rc1, and left the other line as is.

I don't mind

Reply:
*shakes fist at obal*  Scrapping and re-running obal.  probably faster at this point.

Reply:
`--version 4.1.0 --prerelease rc1 --release keep` should do what you need

Reply:
local changes to support parallelization of release/scratch builds  temporarily bit me in the but.  But should _really_ be good now.